src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -56,6 +56,10 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <!-- TODO: remove this P2P reference once new packages are produced and updated -->
```

Comment:
I believe there is an issue open for removing all P2P references with these TODO labels. Is that correct @joperezr?